docs/design/management_plane.md

```diff
@@ -291,14 +291,30 @@ For example:
 9. The Endpoint Operator sees that the Endpoint's state is now provisioned.
 10. The Endpoint Operator updates its own state store with the newly provisioned Endpoint.
 
+### Scaled Endpoint Object Create Workflow
+![Scaled_Endpoint](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/futurewei-cloud/mizar/dev-next/docs/design/puml/create_scaled_endpoint_workflow.puml)
+
+1. The Endpoint Operator is notified that a Scaled Endpoint has been created
+2. The Endpoint Operator allocates an IP, mac, etc, for the new Scaled Endpoint Object
+3. CNI Service is informed about the newly created Scaled Endpoint
+4. The CNI Services call backs to the CNI about the new Scaled Endpoint
+5. The Scaled Endpoint object is updated with Endpoint Objects as its backends
```

Comment:
We will need a separate workflow description for backend update.

Reply:
I have split the workflows. One for create scaled endpoint, and one for update scaled endpoint